scripts/praise.coffee

```diff
@@ -73,7 +74,7 @@ module.exports = (robot) ->
       return
 
     # Start praising/shaming
-    sendMessage(msg, user, shame)
+    sendMessage(msg, user, shame, reason)
     return if --num == 0
     interval = setInterval ->
       sendMessage(msg, user, shame)
```

Comment:
Did you mean to skip the reason here? Seems like you'd want to repeat it over and over again